pkg/module/go_runtime.go

```diff
@@ -0,0 +1,67 @@
+package module
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+// PrepareEnv will return all the appropriate
+// environment variables for a Go Command to run
+// successfully (such as GOPATH, GOCACHE, PATH etc)
+func PrepareEnv(gopath, goProxy string) []string {
```

Comment:
let's make this function private: `preparEnv` now that no one is using it outside this package AFAIK.